js/common/modules/@arangodb/aql/explainer.js

```diff
@@ -1169,7 +1164,7 @@ function processQuery(query, explain, planIndex) {
     var boundValue = bound.isConstant ? value(JSON.stringify(bound.bound)) : buildExpression(bound.bound);
     return attribute(attr) + ' ' + operators[bound.include ? 1 : 0] + ' ' + boundValue;
   };
-
+/*
```

Comment:
We should remove the commented-out code for good, but we can do this in a separate PR if you prefer.